blocks/filter-group/filter-group-functions.js

```diff
@@ -1,66 +1,67 @@
 /**
  * @file Filter group functionality, including event listener logic.
  */
-
-import { dataStore } from '../../scripts/dataStore.js';
 import { updateFilter, getCurrentFiltersArray } from './filter-group-utils.js';
-import { applyTemporaryMockupTags } from './filter-group-temp-mockup.js';
+import { fetchAndBuildIdeas } from '../../scripts/helpers/fetchAndBuildIdeas.js';
+import { initialMaxIdeas, calculateGroupTotal, rearrangeFeatures, updateLoadButtonState } from '../ideas/ideas-functions.js';
+import { debounce } from '../../scripts/helpers/debounce.js';
 
 /**
- * Remove filter buttons for tags that are not assigned to any articles.
+ * Update all articles displayed on the page with ones that have tags
+ * that match the currently selected filters.
  * 
- * @param {parentElement} parentElement Look for the filter buttons within this element. Usually the filter group.
+ * @param {string[]} selectedFilters Show articles with these filter/tag name(s)
  */
-export const removeUnusedTags = async (parentElement) => {
```

Comment:
I got rid of this since the filters are customized via content entry and we now have a null state. It was a bit unnecessary and could cause a button to briefly appear and then disappear.